### DISCUSSION TITLE
购买两年了，有一年上不去。你看我这问题怎么解决一下？

### DISCUSSION THREAD

**Discussion by Reptile024** [getlantern/lantern-forum-cn#1017] [General]
![Screenshot_2024-11-03-06-13-18-076_org getlantern lantern](https://github.com/user-attachments/assets/e84f157a-fca3-4316-93af-730b99675335)
![Screenshot_2024-11-03-06-13-31-320_org getlantern lantern](https://github.com/user-attachments/assets/495b02d8-05fe-4148-b22d-474ed51f66d9)
![Screenshot_2024-11-03-06-13-39-741_org getlantern lantern](https://github.com/user-attachments/assets/ba639061-1dc7-4211-b58e-c978c549422c)
![Screenshot_2024-11-03-06-13-58-963_org getlantern lantern](https://github.com/user-attachments/assets/3e84af4d-7a50-4afb-a3be-5e497cbb2ed3)
![Screenshot_2024-11-03-06-14-09-709_com android browser](https://github.com/user-attachments/assets/eb0f0855-b58d-440a-9ccd-795a7eca1f79)

**Comment by Reptile024** [comment]
![Screenshot_2024-11-03-06-49-53-826_org getlantern lantern](https://github.com/user-attachments/assets/daedbd85-aae7-4fbc-99f1-311448483e6e)

**Comment by MaxMKMax** [comment]
@Reptile024 我会再次帮您反馈. 非常抱歉啊.
